scripts/run-app.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env sh
+
+log() {
+  local message=$1
+  local iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
+  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$1"'"}'
+}
+
+export_secrets_from_dir() {
+  local secrets_dir=$1
+
+  for filename in $secrets_dir/*; do
+    local filename_without_path=${filename##*/}
+    local filename_uppercased=$(echo $filename_without_path | tr 'a-z' 'A-Z');
+    local filename_with_underscores=${filename_uppercased//-/_}
+
+    local secret_name="SECRET_$filename_with_underscores";
+
+    log "Exporting secret '$filename_without_path' as '$secret_name'"
+
+    local secret=$(<$filename);
+
+    export "$secret_name=$secret"
+  done
+}
+
+start_app() {
+  # npm provides binaries on PATH in scripts, so we have to do the same
+  export PATH=$PATH:$(pwd)/node_modules/.bin
+
+  local start_script=$(node -p "require('./package.json').scripts.start")
```

Comment:
Why not just `npm start`?

Reply:
we remove `npm` after installing the packages for a smaller image size 😄

Reply:
Is it really that much to save on doing that?

Reply:
/shrug. I don't want to change it for this feature :)

Reply:
npm/yarn is over 20% of the image

Reply:
I have zero issues with removing `yarn`, but to be honest I do have doubts about removing `npm`. `npm` is shipped with node and is a part of node. Due to that I would actually expect `npm` to be pressent in the runtime for the whole life time of the process.

Reply:
I suppose this is relevant for the cases where people do "npm run <something>" as part of their scripts, but that doesn't really change with this PR (and isn't very useful when you're just starting the app -- it's much more common for linting and building). Maybe we should support "prestart" though?

Reply:
nah, I don't think it's needed.